app/main.py

```diff
@@ -97,6 +97,8 @@ def __init__(self, keys):
     def process(self, encrypted_jwt, tx_id=None):
 
         self.bound_logger = self.bound_logger.bind(tx_id=tx_id)
+        self.bound_logger = self.bound_logger.try_unbind("survey_id")
```

Comment:
The unbind should happen at the end of this function, not at the start.  Here is an example https://github.com/ONSdigital/sdx-collect/blob/master/app/response_processor.py#L88

Reply:
Yep, that's a fair cop